app/assets/javascripts/admin/addon/components/admin-flags.gjs

```diff
@@ -46,19 +48,44 @@ export default class AdminFlags extends Component {
     });
   }
 
+  @action
+  deleteFlagCallback(flag) {
+    return ajax(`/admin/config/flags/${flag.id}`, {
+      type: "DELETE",
+    })
+      .then(() => {
+        this.flags.removeObject(flag);
+      })
+      .catch((error) => {
+        return popupAjaxError(error);
+      });
+  }
+
   <template>
     <div class="container admin-flags">
-      <h1>{{i18n "admin.flags.title"}}</h1>
-      <table class="flags grid">
+      <div class="admin-flags__header">
```

Comment:
I think after we add a couple more config areas we should make a reusable component for combination of title + help text + button(s)